CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ find_package(ament_cmake_vendor_package REQUIRED)
 
 ament_vendor(mimick_vendor
   VCS_URL https://github.com/ros2/Mimick.git
```

Comment:
Since you did https://github.com/ros2/Mimick/pull/38 from a fork, for the purposes of this test I think you need to change this to https://github.com/ahcorde/Mimick (this will obviously have to be undone before we merge).

Reply:
It was able to find the hash https://ci.ros2.org/job/ci_linux/21543/consoleFull

```
22:22:24 From https://github.com/ros2/Mimick
22:22:24  * branch            d6ed6a70535cf1a2d1f5d9665ae66065ffda7f44 -> FETCH_HEAD
22:22:24 Note: switching to 'd6ed6a70535cf1a2d1f5d9665ae66065ffda7f44'.
22:22:24 
22:22:24 You are in 'detached HEAD' state. You can look around, make experimental
22:22:24 changes and commit them, and you can discard any commits you make in this
22:22:24 state without impacting any branches by switching back to a branch.
22:22:24 
22:22:24 If you want to create a new branch to retain commits you create, you may
22:22:24 do so (now or later) by using -c with the switch command. Example:
22:22:24 
22:22:24   git switch -c <new-branch-name>
22:22:24 
22:22:24 Or undo this operation with:
22:22:24 
22:22:24   git switch -
22:22:24 
22:22:24 Turn off this advice by setting config variable advice.detachedHead to false
22:22:24 
22:22:24 HEAD is now at d6ed6a7 Added Windows path to CreateToolhelp32Snapshot
```